queries.sql

```diff
@@ -50,27 +50,27 @@ WHERE weight_kg between 10.4 and 17.3;
 Verify that change was made. Then roll back the change and verify that species columns went back to the state before transaction.
 */
 
-BEGIN TRANSACTION;
+-- BEGIN TRANSACTION;
 
-UPDATE animals SET species = 'unspecified'; 
+-- UPDATE animals SET species = 'unspecified'; 
 
-SELECT * FROM animals;
+-- SELECT * FROM animals;
 
-ROLLBACK TRANSACTION;
+-- ROLLBACK TRANSACTION;
 
-SELECT * FROM animals;
+-- SELECT * FROM animals;
 
 
 /*Update species based on name*/
 
-BEGIN TRANSACTION;
+-- BEGIN TRANSACTION;
 
-UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon'; 
-UPDATE animals SET species = 'pokemon' WHERE species IS NULL; 
+-- UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon'; 
+-- UPDATE animals SET species = 'pokemon' WHERE species IS NULL; 
 
-COMMIT TRANSACTION;
+-- COMMIT TRANSACTION;
 
-SELECT * FROM animals;
+-- SELECT * FROM animals;

```

Comment:
- [ ] (OPTIONAL) This is not really optional. Kindly remove all commented code in order to keep your work neat 👍